src/Makefile.am

```diff
@@ -1,8 +1,9 @@
 polycapincludedir = $(includedir)/polycap
 polycapinclude_HEADERS = polycap.h
```

Comment:
Zou er in deze Makefile mss een probleem zitten? Kheb de indruk dat ik bij compilatie veel errors krijg wanneer ik verwijs naar polycap_profile*, en dat hij dit doet omdat hij mss include/polycap.h niet juist herkent. Of is dit zeker juist? (in welk geval ik de polycap_profile structure mss nog in detail moet beschrijven in polycap-private.h)

Reply:
Zal eens kijken